websocket/polygon.go

```diff
@@ -125,12 +129,34 @@ func getParams(market Market, topic Topic, tickers ...string) (string, error) {
 	return strings.Join(params, ","), nil
 }
 
+func (c *Client) setSubscription(prefix string, ticker string) {
```

Comment:
i think these should've used variadic args: `setSubscription(topic Topic, tickers ...string)`

Reply:
I could change the implementation, but as it stands setSubscription only takes in one ticker at a time so we wouldn't need tickers ...string. As for my use of prefix over topic, I did that because the parameter is taking in topic.prefix(). I thought it would be better to be more specific.

Reply:
i'm suggesting that you should change the implementation. topic is more descriptive than prefix imo (you subscribe to a topic, not a prefix).

Reply:
no, the helper is fine but i don't think you should force users of this func to loop over variadic args when they could pass them to the helper instead (similar to `getParams(...)`)